backend/YouTubeV2/Controllers/SubscriptionsController.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc;
+using YouTubeV2.Application.DTO;
+using YouTubeV2.Application.Services;
+
+namespace YouTubeV2.Api.Controllers
+{
+	[ApiController]
+	public class SubscriptionsController : ControllerBase
+	{
+		private readonly SubscriptionsService _subscriptionsService;
+
+		public SubscriptionsController(SubscriptionsService subscriptionsService)
+		{
+			_subscriptionsService = subscriptionsService;
+		}
+
+		[HttpGet("subscriptions")]
+		public async Task<ActionResult<IReadOnlyList<SubscriptionDTO>>> GetSubscriptionsAsync([FromQuery] string id, CancellationToken cancellationToken)
```

Comment:
i would add `[Required]` to id. Also i think it should be guid not string (v1.0.4)

Reply:
Added and changed